python/ECCEmailer.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+
+#
+# Send an automated email based on provided information.
+#
+# This script presumes that smtp-relay.gmail.com has been setup to
+# accept emails from this public IP address without authentication.
+# To set this up:
+#
+# 1. Login to admin.google.com as an administrator.
+# 2. Apps
+# 3. G Suite
+# 4. Gmail
+# 5. Advanced settings (at the bottom of the list)
+# 6. Scroll down to Routing and find SMTP relay service
+# 7. Add the desired IP address:
+#    - require TLS encryption: yes
+#    - require SMTP authentication: no
+#    - allowed senders: Only addresses in my domains
+#
+# And then make sure that the sender is actually a valid email address
+# in the ECC Google domain.
+#
+
+
+import smtplib
+import argparse
+
+from email.message import EmailMessage
+
+import ECC
+
+app_cred_file = 'client_id.json'
+default_user_cred_file = 'user-credentials.json'
+user_agent = 'gxcopy'
+
+smtp_server 	= 'smtp-relay.gmail.com'
+smtp_from   	= 'Epiphany <no-reply@epiphanycatholicchurch.org>'
```

Comment:
The name component of this default "from" email address should probably be "Epiphany Catholic Church".